resources/js/Pages/Project/Members/Index.vue

```diff
@@ -290,7 +290,6 @@ export default {
   },
 
   created() {
```

Comment:
The difference is big: 

* `created()` : called before the DOM has been mounted or rendered.
* `mounted()` : it’s called after the DOM has been mounted or rendered. 

So here we need to initialize our objects before the DOM is mounted, otherwise we will have errors since the view builds elements that is supposed to be in the DOM (here, a list).